### DISCUSSION TITLE
View packets or the inbound traffic

### DISCUSSION THREAD

**Discussion by cerberus10748** [Security-Onion-Solutions/securityonion#14057] [General]
Hello, 
I have an issue where i want to verify if a sensor actually has data flow on the monitor port. Is there a command in ssh i can run to basically see the packets and their description on the cli as they come in versus on the Grid status.

**Comment by dougburks** [comment]
You can use tcpdump like this:
```
sudo tcpdump -nni bond0
```

For more information, please see:
https://www.tcpdump.org/manpages/tcpdump.1.html
